### PR TITLE
fix generated columns in mutable columns for PostgreSQL

### PR DESCRIPTION
**Pull request by rangzen**
Fix https://github.com/go-jet/jet/issues/209

### CODE REVIEW

**Comment by rangzen** [comment]
Need data from https://github.com/go-jet/jet-test-data/pull/3

File: tests/postgres/generated_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package postgres
+
+import (
+	. "github.com/go-jet/jet/v2/tests/.gentestdata/jetdb/test_generated/table"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
```
Comment (go-jet) [inline tests/postgres/generated_test.go:8]: This tests are intended to be used with a live db. For generated column tests we can write a tests to test simple SELECT, INSERT or UPDATE query. You can then test mutable columns by comparing `.Sql()` or `.DebugSql()` query using `testutils.AssertStatementSql` or `testutils.AssertDebugStatementSql`.  For example - [check](https://github.com/go-jet/jet/blob/ce3c622fa4f8b6f7a18a8c84a386f710e6bef430/tests/postgres/sample_test.go#L17).
Result of the db query is not of importance, just error check would be enough.

The test can be added in existing `sample_test.go`.

Comment (rangzen) [reply tests/postgres/generated_test.go]: SQL and test migrated and updated.

**Comment by go-jet** [comment]
LGTM. I've merged test data PR. To fix this PR build you'll need to push one more commit, with updated sub-module master head. You can just call `make checkout-testdata` and then push changes.

**Comment by rangzen** [comment]
CircleCI didn't take the new link to jet-test-data. Did I misse something?

**Comment by rangzen** [comment]
I cannot restart the CI myself.

**Comment by go-jet** [comment]
Ahh, sorry I missed this. `postgres` tests are run on two databases `postgres` and `cockrouchdb`. pg tests are successfully, but cockrochdb are failing. To fix the tests we will have to add a the same table to [cockroach test init data](https://github.com/go-jet/jet-test-data/blob/master/init/cockroach/test_sample.sql).

**Comment by rangzen** [comment]
https://github.com/go-jet/jet-test-data/pull/4

File: generator/postgres/query_set.go [hunk 2]
```diff
@@ -44,6 +44,7 @@ WITH primaryKeys AS (
 )
 SELECT column_name as "column.Name", 
 	   is_nullable = 'YES' as "column.isNullable",
+	   is_generated = 'ALWAYS' as "column.isGenerated",
```
Comment (go-jet) [inline generator/postgres/query_set.go:47]: We will need to update this condition since `cockroachdb` doesn't use `ALWAYS` but `YES` as a constant.

Comment (rangzen) [reply generator/postgres/query_set.go:47]: Do you have an example in the code where we send one way for PostgreSQL and another way for CockroachDB?

Comment (go-jet) [reply generator/postgres/query_set.go:47]: This query is used for both databases. Cockroach shares interfaces with postgres, but there are occasionally some differences. So we just have to add: `is_generated = 'ALWAYS' or is_generated = 'YES'`

Comment (rangzen) [reply generator/postgres/query_set.go]: Seems ok in local tests.

Comment (go-jet) [reply generator/postgres/query_set.go]: Yep, looks good.

**Comment by rangzen** [comment]
@go-jet Thank you 🥳

**Pull request merged**